### PR TITLE
WIP: v2.0 documentation

### PR DESCRIPTION
**Pull request by klim0v**

### CODE REVIEW

File: content/docs/index.md [hunk 1]
```diff
@@ -684,6 +686,163 @@ type EditCandidatePublicKeyData struct {
...
+To create liquidity through this pool.
+
+*Data field contents:*
+
+```go
+type AddSwapPoolData struct {
+	Coin0      uin32
+	Coin1      uin32
+	Volume0    *big.Int
+	MaxVolume1 *big.Int
```
Comment (shrpne) [inline content/docs/index.md:705]: MaximumVolume1

File: content/docs/index.md [hunk 2]
```diff
@@ -668,7 +670,7 @@ type PriceVoteData struct {
 
 Type: **0x14**
 
-Transaction to change candaite public key.
```
Comment (shrpne) [inline content/docs/index.md:671]: перенос строки

File: content/docs/index.md [hunk 3]
```diff
@@ -684,6 +686,163 @@ type EditCandidatePublicKeyData struct {
...
+
+When a new liquidity provider deposits tokens into an existing Uniswap pair, the number
+of liquidity tokens minted is computed based on the existing quantity of tokens:
+
+![](https://i.ibb.co/YkQdMLd/image.png)
+
+if they are the first depositor, the number of liquidity tokens equal to the geometric mean
+of the amounts deposited:
+
+![](https://i.ibb.co/cFXTdfQ/image.png)
```
Comment (klim0v) [inline content/docs/index.md:722]: volume0 and volume1

File: content/docs/index.md [hunk 4]
```diff
@@ -557,16 +564,11 @@ type EditCandidateData struct {
...
 
-
-
-
-
-### Since Minter 1.2 released, there are few new transactions added:
 ### Set halt block transaction
 Type: **0x0F**
 
-Since Minter 1.2 released, validators can now vote for a particular block to stop the blockchain for update.
```
[the remark this answers is not in the document]
Comment (klim0v) [reply content/docs/index.md]: resolve

File: content/docs/index.md (hunk 1 above)
Comment (klim0v) [reply content/docs/index.md]: done

File: content/docs/index.md (hunk 2 above)
Comment (klim0v) [reply content/docs/index.md]: ok

**Comment by klim0v** [comment]
todo: https://gist.github.com/klim0v/eab8172f0ffc73b1bcf9fe63746f9b4c

**Pull request merged**